### ISSUE TITLE
Invalid internal links for "spellcheck" page during build

### ISSUE THREAD

**Issue by f1ames** [ckeditor/ckeditor4#3178]
## Type of report

Docs

## Provide a description of requested docs changes

After introducing changes from #2992, building docs will throw an `Invalid internal links` error:

![image](https://user-images.githubusercontent.com/1061942/59671196-8d7aea00-91bd-11e9-9d39-59ba3a1b923f.png)

It is caused by the fact that we have moved `guide/dev_spellcheck.html` subsection to new section `features/spellcheck.html` and there are still links in SCAYT/WSC plugins linking to the old URLs. We will be preparing PRs to both to cover that, but we have to live with that for some time.

As there will be proper redirect from old URLs to new ones it won't affect docs usability in any way.

---

To sum up, the proper resolution of this issue is to prepare PRs to both WSC and SCAYT plugins with links changes and when they get accepted we need to update plugins in `ckeditor-presets`.

**Comment by engineering-this** [comment]
https://github.com/WebSpellChecker/ckeditor-plugin-scayt/pull/169 is merged, we can change submodule branch to current `master` in `ckeditor-presets` or wait for release and then bump submodule version.

**Comment by f1ames** [comment]
Fixed in https://github.com/ckeditor/ckeditor4-presets/commit/f08993644ed6431ad7bf998ef8b5f769feeece72.
